src/lib/util/debug.c

```diff
@@ -159,7 +159,8 @@ char const CC_HINT(used) *__lsan_default_suppressions(void)
 		"leak:*gmtsub*\n"
 		"leak:tzsetwall_basic\n"
 		"leak:ImageLoaderMachO::doImageInit\n"
-		"leak:libSystem_atfork_child"
+		"leak:libSystem_atfork_child\n"
+		"leak:getaddrinfo"
```

Comment:
Remove the NAT entry too... It's the same code path.

Reply:
done